ush/job_preamble.sh

```diff
@@ -121,16 +121,16 @@ export -f POST_STEP
 #
 #-----------------------------------------------------------------------
 #
+[[ "$WORKFLOW_MANAGER" = "rocoto" ]] && export COMROOT=$COMROOT
 if [ "${RUN_ENVIR}" = "nco" ]; then
-    export COMROOT=$COMROOT
     export COMIN="${COMIN:-$(compath.py -o ${NET}/${model_ver}/${RUN}.${PDY}/${cyc})}"
     export COMOUT="${COMOUT:-$(compath.py -o ${NET}/${model_ver}/${RUN}.${PDY}/${cyc})}"
     export COMINm1="${COMINm1:-$(compath.py -o ${NET}/${model_ver}/${RUN}.${PDYm1})}"
   
     export COMINgfs="${COMINgfs:-$(compath.py ${envir}/gfs/${gfs_ver})}"
     export COMINgefs="${COMINgefs:-$(compath.py ${envir}/gefs/${gefs_ver})}"
 
-    export KEEPDATA="${KEEPDATA:-$KEEPDATA}"
+#    export KEEPDATA="${KEEPDATA:-$KEEPDATA}"
```

Comment:
Not necessary

Reply:
It is wrong to code the KEEPDATA assignment with two identical value.

Reply:
I don't see any purpose for this line - if KEEPDATA isn't defined, it won't help anything.